app/Transformers/UserCompactTransformer.php

```diff
@@ -70,6 +72,11 @@ public function includeCover(User $user)
         });
     }
 
+    public function includeCurrentModeRank(User $user)
+    {
+        return $this->primitive($user->statistics(Auth::user()->playmode)->rank ?? false);
```

Comment:
- check `Auth::user()` isn't null. Also maybe use `auth()` instead of `Auth` to skip an import
- `null` instead of `false`
- `rank` is an old unused column; use `globalRank()` instead

Reply:
hmm other transformers are using an import but i can use ``auth()`` if it's really a deal ¯\\\_(ツ)_/¯

Reply:
newer codes generally use `auth()`, I think.